validate_great_expectations/item.yaml

```
@@ -17,10 +17,10 @@ mlrunVersion: 1.1.0
 name: great-expectations
```

Name of the function need to be the same as the filename
```suggestion
name: validate-great-expectations
```

---

Name of the function has been changed to `validate_great_expectations`. All file names have been changed accordingly

---

This was to prevent clashes between the function and the underlying Python library

---

Ok I see where you are talking about. Will make the changes